client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getPaperSizes } from '../../lib/pdf-label-utils';
+import Button from 'components/button';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import PaymentMethod, { getPaymentMethodTitle } from './label-payment-method';
+
+class ShippingLabels extends Component {
+	componentWillMount() {
+		this.setState( { expanded: this.isExpanded( this.props ) } );
+	}
+
+	componentWillReceiveProps( props ) {
+		this.setState( { expanded: this.isExpanded( props ) } );
+	}
+
+	isExpanded( { pristine, selectedPaymentMethod } ) {
+		return ! selectedPaymentMethod || ! pristine;
+	}
+
+	renderPlaceholder() {
+		return (
+			<div className="label-settings__placeholder">
+				<FormFieldSet>
+					<FormLabel className="label-settings__cards-label">
+						<span />
+					</FormLabel>
+					<FormSelect />
+				</FormFieldSet>
+				<FormFieldSet>
+					<FormLabel
+						className="label-settings__cards-label">
+						<span />
+					</FormLabel>
+					<p className="label-settings__credit-card-description">
+					</p>
+					<PaymentMethod selected={ false } isLoading={ true } />
+					<PaymentMethod selected={ false } isLoading={ true } />
+					<Button href="https://wordpress.com/me/billing" target="_blank" compact />
+				</FormFieldSet>
+			</div>
+		);
+	}
+
+	renderPaymentsSection = () => {
+		const { paymentMethods, setFormDataValue, selectedPaymentMethod, translate } = this.props;
+
+		if ( ! this.state.expanded ) {
+			const expand = ( event ) => {
+				event.preventDefault();
+				this.setState( { expanded: true } );
+			};
+
+			const {
+				card_type: selectedType,
+				card_digits: selectedDigits,
+			} = find( paymentMethods, { payment_method_id: selectedPaymentMethod } );
+
+			return (
+				<div>
+					<p className="label-settings__credit-card-description">
+						{ translate( 'We\'ll charge the credit card on your ' +
+							'account (%(card)s) to pay for the labels you print', { args: {
+								card: getPaymentMethodTitle( translate, selectedType, selectedDigits ),
+							} } ) }
+					</p>
+					<p className="label-settings__credit-card-description">
+						<a href="#" onClick={ expand }>{ translate( 'Choose a different card' ) }</a>
+					</p>
+				</div>
+			);
+		}
+
+		const onPaymentMethodChange = ( value ) => setFormDataValue( 'selected_payment_method_id', value );
+
+		let description, buttonLabel;
+		if ( paymentMethods.length ) {
+			description = translate( 'Use your credit card on file to pay for the labels you print or add a new one.' );
+			buttonLabel = translate( 'Add another credit card' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Add Credit Card' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).